doc/source/whatsnew/v1.0.1.rst

```diff
@@ -15,7 +15,7 @@ including other versions of pandas.
 
 Bug fixes
 ~~~~~~~~~
-
+- bug ``Groupby.apply`` was returning ``TypeError`` if called with function which returned list (:issue:`31441`)
```

Comment:
capitalize. use a reference for Groupby.applyl (look at other whatsnew to see how to get this reference).